dev/io.openliberty.org.jboss.resteasy.common.ee10/test/io/openliberty/org/jboss/resteasy/common/core/se/LibertyResteasyJavaSeBootstrapTest.java

```diff
@@ -0,0 +1,109 @@
+
+import org.junit.Test;
+
+import jakarta.ws.rs.ApplicationPath;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.SeBootstrap;
+import jakarta.ws.rs.core.Application;
+
+/**
+ * Unit tests to ensure an attempt to use the optional Java Se Bootsrap APIs in Liberty will result in
+ * an exception.
+ */
+public class LibertyResteasyJavaSeBootstrapTest {
+
+    @Test
+    public void testAttemptToUseJavaSEBootstrapAPIThrowsException() {
+        boolean exceptionCaught = false;
+        try {
+            final SeBootstrap.Configuration.Builder bootstrapConfigurationBuilder = SeBootstrap.Configuration.builder();
+        } catch (Throwable t) {
+            exceptionCaught = true;
+            assertTrue("Received throwable message is: " + t.getMessage() + " and the expected message is: Liberty does not support the optional Jakarta Rest SE Bootstrap API.",
+                       t.getMessage().contains("Liberty does not support the optional Jakarta Rest Java SE Bootstrap API."));
+        }
+        if (!exceptionCaught) {
+            fail("Expected UnsupportedOperationException not caught");
+        }
+    }
+    @Test
+    public void testAttemptToStartJavaSeBootstrapAPIThrowsException() {
+        boolean exceptionCaught = false;
+        try {
+            final Application application = new TestApplication();
+            final CompletionStage<SeBootstrap.Instance> completionStage = SeBootstrap.start(application,
+                null);
+        } catch (Throwable t) {
+            exceptionCaught = true;
+            assertTrue("Received throwable message is: " + t.getMessage() + " and the expected message is: Liberty does not support the optional Jakarta Rest SE Bootstrap API.",
+                       t.getMessage().contains("Liberty does not support the optional Jakarta Rest Java SE Bootstrap API."));
+        }
+        if (!exceptionCaught) {
+            fail("Expected UnsupportedOperationException not caught");
+        }
+    }
+    @Test
+    public void testAttemptToStartJavaSeBootstrapAPIWithClassThrowsException() {
+        boolean exceptionCaught = false;
+        try {
+            final Application application = new TestApplication();
+            final CompletionStage<SeBootstrap.Instance> completionStage = SeBootstrap.start(application.getClass(),
+                null);
+        } catch (Throwable t) {
+            exceptionCaught = true;
+            assertTrue("Received throwable message is: " + t.getMessage() + " and the expected message is: Liberty does not support the optional Jakarta Rest SE Bootstrap API.",
+                       t.getMessage().contains("Liberty does not support the optional Jakarta Rest Java SE Bootstrap API."));
+        }
+        if (!exceptionCaught) {
+            fail("Expected UnsupportedOperationException not caught");
+        }
+    }
+ 
+    @ApplicationPath("test")
+    public static final class TestApplication extends Application {
```

Comment:
```suggestion
    public static class TestApplication extends Application {
```
We just had a challenge about this.

Reply:
That challenge involved CDI, but I agree and will remove.